### ISSUE TITLE
Can't remove package in workspace based project

### ISSUE THREAD

**Issue by donaldpipowitch** [yarnpkg/yarn#3972]
**Do you want to request a *feature* or report a *bug*?**

*bug*

**What is the current behavior?**

```
$ yarn remove react
yarn remove v0.27.5
error No lockfile in this directory. Run `yarn install` to generate one.
```

**If the current behavior is a bug, please provide the steps to reproduce.**

0. `$ git clone git@github.com:pedronauck/yarn-workspaces-example.git`
0. `$ cd yarn-workspaces-example/packages/a`
0. `$ yarn add react`
0. `$ yarn remove react`

**What is the expected behavior?**

- removes `react` as a dependency in `./packages/a/package.json`
- updates `./yarn.lock`
- updates `./node_modules/` and `./packages/a/node_modules/`

**Please mention your node.js, yarn and operating system version.**

- Node v6.9.1
- yarn v0.27.5
- Mac OS X

--- 

Workspaces look great so far! ♥

**Comment by edmorley** [comment]
Hi! I think this is a (forward) dupe of #4334 which is meant to be fixed by #4654, which is in the recently released yarn 1.2.0. Could you give the new version a try? :-)

**Comment by donaldpipowitch** [comment]
This works 😍  Thank you.
